### PR TITLE
Adding some Rails templates

### PR DESCRIPTION
**Pull request by arfon**
Still working on the new front-page template.

### CODE REVIEW

File: app/views/layouts/temp_home.html.erb [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-    <title>The Open Journal</title>
+    <title>The Open Journal of Astrophysics</title>
 
     <%= javascript_include_tag "static" %>
     <%= stylesheet_link_tag "static", media: "all" %>
 
-    <link type="text/css" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,500,700,">
-    <link type="text/css" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Neuton:400,700">
+    <link type="text/css" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:200,300,400,500,700,">
+    <link type="text/css" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Neuton:200,300,400,700">
```
Comment (arfon) [inline app/views/layouts/temp_home.html.erb]: Added a couple of new weights here to better match the designs.

File: app/controllers/home_controller.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@ def index
   end
 
   def temp_home
-    render :temp_home, :layout => 'temp_home'
+    @papers = Paper.published
```
Comment (arfon) [inline app/controllers/home_controller.rb]: We should paginate here eventually.

File: app/controllers/home_controller.rb [hunk 3]
```diff
@@ -4,6 +4,7 @@ def index
...
+    render :temp_home, :layout => 'temp_home', :papers => @papers
```
Comment (marcrohloff) [inline app/controllers/home_controller.rb]: Can you use the new hash syntax here?

**Comment by marcrohloff** [comment]
Tell me when you're at a point that you want to merge. We can also roll back the previous PR if you prefer.

**Pull request merged**